app/addons/cors/components/OriginTable.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import OriginRow from "./OriginRow";
+
+export default class OriginTable extends Component {
+
+  constructor (props) {
+    super(props);
+  }
+
+  createRows () {
+    return _.map(this.props.origins, function (origin, i) {
```

Comment:
if you change that function to `(origin, i) => {...},` You don't need to pass in the `this` a the end of the map